ds_judgements_public_ui/templates/includes/gtm/gtm_body.html

```diff
@@ -1,6 +1,8 @@
-<!-- Google Tag Manager (noscript) -->
-<noscript>
-    <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-MMVM4CJ"
-            height="0" width="0" style="display:none;visibility:hidden"></iframe>
-</noscript>
-<!-- End Google Tag Manager (noscript) -->
+{% if showGTM %}
```

Comment:
Hi @LewisDaleUK - would it be OK to add a similar `if` condition to the `gtm_head.html` file?